### DISCUSSION TITLE
Failed to start gauge API: Timed out connecting to java

### DISCUSSION THREAD

**Discussion by berhangunes** [getgauge/gauge#2192] [General]
Hi, I got an error when I try to run specs in maven project. Intellij version is 2020.3.4. 

> Installed version of gauge-java (0.7.13) does not match with dependency gauge-java (0.6.0) specified in pom.xml file
Error ----------------------------------
[Gauge]
Failed to start gauge API: Timed out connecting to java

Gauge version: 1.0.7
Commit Hash: ed7b4fd6

Plugins
-------
html-report (4.0.12)
java (0.7.13)
screenshot (0.0.1)
spectacle (0.1.3)
xml-report (0.2.3)

When I run the same project with the same versions of gauge, gauge plugins and intellij in a different computer it runs the specs.

**Comment by sriv** [comment]
the error seems to indicate that the project has an older version of gauge-java as a dependency in `pom.xml`. Could you please check?

**Reply by berhangunes** [reply]
I changed the dependency gauge-java to 0.7.13 but now it gives me Step Implementation not found error.
